test_cases/assert_violations/constructor_create.yaml

```diff
@@ -1,8 +1,9 @@
-description: "Assert violation after call to concretely created contract"
+description: Assert violation after call to concretely created contract
 issues:
-- id: "SWC-110"
+- id: SWC-110
   count: 1
   locations:
-  - bytecode_offsets: [277]
-    line_numbers: [12]
-
+  - bytecode_offsets:
+      '0xfd73371b884f20a50acfd8f790496e905a7c397d16cf5a30dd5fcb461499d470': [277]
```

Comment:
I think this should be the following instead:
295:1:0 (0xc27f0d2e53876fb4e1100569b750ff701f48b85487e89782f8b8e4dd72369e67)